src/SBVRType.d.ts

```diff
@@ -0,0 +1,44 @@
+interface SBVRType<I,O> {
```

Comment:
This file should be redundant - typescript can get all the types from the .ts files instead and you can point a `types` entry in package.json at the .ts file directly :)